diff --git a/src/drawcommandbuffer.c b/src/drawcommandbuffer.c
index 1f3c309..63b53c1 100644
--- a/src/drawcommandbuffer.c
+++ b/src/drawcommandbuffer.c
@@ -1747,7 +1747,7 @@ void sceGnmDrawCmdEventWriteEop(
 
 	uint32_t sel = EOP_DST_SEL(EOP_DST_SEL_MEM) | EOP_DATA_SEL(datasel);
 	if (datasel != GNM_DATA_SEL_DISCARD)
-		sel |= EOP_INT_SEL(EOP_INT_SEL_SEND_DATA_AFTER_WR_CONFIRM);
+		sel |= EOP_INT_SEL(2); /* interrupt after write confirm */
 
 	cmd->cmdptr[0] = PKT3(PKT3_EVENT_WRITE_EOP, 4, 0);
 	cmd->cmdptr[1] =
